### ISSUE TITLE
Car bluetooth call events

### ISSUE THREAD

**Issue by basulle** [BelledonneCommunications/linphone-android#2194]
Hi, @Viish 

There is a question, are you handling anyhow car's Bluetooth calls events? I mean hang up, mute etc?
I need to pay at least for your dinner for all the help provided :D

Thank you!

**Comment by Viish** [comment]
HI @basulle,

You can do it quite easily using https://developer.android.com/reference/android/telecom/TelecomManager APIs (or https://developer.android.com/jetpack/androidx/releases/core#core-telecom-1.0.0-alpha03).

Cheers,

**Comment by basulle** [comment]
Hi, @Viish 
Having some difficulties with implementing this.
I have created inCallService with onCallAdded method to detect the call, then I have created ConnectionService with own Connection inherited from Connection to handle the events.
Then I am trying to call telecomManager.addNewIncoming call on IncomingReceived, but the app is crashing because of PhoneAccount, which I need to solve.

So the question is am I doing everything correct, or I need to provide TelecomManager to Linphone so it will handle something for that?

Maybe you have some example of how this can be achieved, will be so appreciated. 

Thank you!
